theano/tensor/tests/test_basic.py

```diff
@@ -8027,7 +8027,7 @@ def test_numpy_compare_tuple(self):
         b = tensor.tensor3(dtype='float32')
         c = tensor.tensor3(dtype='float32')
 
-        A = numpy.asarray(numpy.random.random_integers(0, 1, (2, 1, 1)),
+        A = numpy.asarray(numpy.random.randint(low=0, high=1, size=(2, 1, 1)),
```

Comment:
This would only sample zeros, which is probably not what we want. `randint(0, 2, size=(...))` should be equivalent to what we had.